### ISSUE TITLE
When using same ID twice and this ID is ROOTID, DAPS message is unhelpful

### ISSUE THREAD

**Issue by ghost** [openSUSE/daps#390]
When the same ID exists twice & that ID is also set as the ROOTID, DAPS returns this:

```
/usr/share/daps/make/setfiles.mk:95: *** Fatal error: ROOTID belongs to an unsupported root element (chapter chapter). Must be one of appendix article bibliography book chapter colophon dedication glossary index part preface refentry reference sect1 section set setindex .  Stop.
```

It would be nice if DAPS could notice at this point that the ROOTID cannot be unique, as it gets two elements back.

**Comment by fsundermeyer** [comment]
I cannot reproduce this - in case an ID appears twice, it does not validate and DAPS correctly errors out:

    > ack -l "id=\"cha.daps.user.layout" xml/
    xml/daps_user_install.xml
    xml/daps_user_layout.xml
    > daps -d DC-daps-user pdf --rootid="cha.daps.user.layout"
    /local/git/daps/doc/build/.profiled/x86_amd64_em64t_osuse_noquick/MAIN.DAPS.xml:5: element   chapter: validity error : ID cha.daps.user.layout already defined
    Document /local/git/daps/doc/build/.profiled/x86_amd64_em64t_osuse_noquick/MAIN.DAPS.xml does not validate
    /usr/share/daps/make/validate.mk:38: recipe for target    '/local/git/daps/doc/build/.profiled/x86_amd64_em64t_osuse_noquick/.validate' failed
    make: *** [/local/git/daps/doc/build/.profiled/x86_amd64_em64t_osuse_noquick/.validate] Error 3

**Comment by ghost** [comment]
Can still reproduce this. The following configuration:
* DocBook 5
* Same ID on two `chapter`s in separate files 
* `gdaps -d DC-SLES-all html --rootid my.duplicate.id` (with development daps @ dc95cec)

(for good measure, i also removed the `build` dir before running.)

**Comment by tomschr** [comment]
Looks like we could check for double IDs in the `daps/daps-xslt/common/rootid.xsl` file.

**Comment by tomschr** [comment]
@fsundermeyer I can reproduce this behaviour that Stefan showed. Used the `doc-ses` docu. I think there is an easy fix for this. These are the respective lines:

```make
# vi +93 /usr/share/daps/make/setfiles.mk
# COUNT contains a number:
COUNT := $(shell $(XMLSTARLET) sel -t -v "count(//*[@id=$ROOTID])" $(SETFILES_TMP) 2>/dev/null)
# FIXME: Check if COUNT is < 2; not sure if this is the right expression:
ifeq $COUNT < 2
 ROOTELEMENT := $(shell $(XMLSTARLET) sel -t -v "//div[@id='$(ROOTID)'][1]/@remap" $(SETFILES_TMP) 2>/dev/null)
  ifneq "$(ROOTELEMENT)" "$(filter $(ROOTELEMENT),$(VALID_ROOTELEMENTS))"
    $(error Fatal error: ROOTID belongs to an unsupported root element ($(ROOTELEMENT)). Must be one of $(VALID_ROOTELEMENTS) )
  endif
else
  $(error Fatal error: discovered one or more elements with the id=$ROOTID)
endif

```

**Comment by fsundermeyer** [comment]
Fixed with 6446e2a
